src/NServiceBus.Hosting.Tests/MyEndpointConfig.cs

```diff
@@ -0,0 +1,12 @@
+namespace EndpointTypeDeterminerTests
+{
+    using NServiceBus;
+
+    class MyEndpointConfig : IConfigureThisEndpoint
+    {
+        public void Customize(EndpointConfiguration configuration)
+        {
+
```

Comment:
It's actually used, see https://github.com/Particular/NServiceBus.Host/pull/57/files/3c1915c978913afc115aacb863ce4248cd738449#diff-f034fb2a8d52cb17d06a8e5992fd4d10R4